.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,40 @@
-## Proposed Changes
+Describe proposed changes and the issue being fixed
 
-  -
-  -
-  -
+<!-- Optional -->
+Fixes [link to the issue]
 
 ## Testing
+<!-- Optional -->
+Describe how you tested your changes. If possible and needed:
+- Test it on a sample project
+- Write unit tests
+- Provide a code snippet
 
-Test: Describe how you tested your changes. Note that this line (with `Test:`) is required, your PR will not build without it!
+<!-- Optional -->
+This should be tested by QA
 
-## Issues Fixed
+## Release Notes
+<!--
+Optional, if omitted - won't be included in the changelog
 
-Fixes: [Optional] The bug on [https://issuetracker.google.com](https://issuetracker.google.com) being fixed
+Sections:
+- Highlights
+- Known issues
+- Breaking changes
+- Features
+- Fixes
+- Prerelease fixes
```

Comment:
What's the difference between prerelease fixes and regular fixes? Is the distinction important?

Reply:
Prelease fixes are fixes of the bugs that we introduced in previous dev/beta release (as a regression, or in a new feature).

We'll include them in the next beta changelog, but won't include in the stable changelog.

I am not sure about adding a separate category for this. What do you think?

There is an option to keep only "Fixes", but add a separate indicator:

```
prerelease fix - add it if it fixes a bug introduced in a previous prerelease version (dev/beta). It will excluded from the stable changelog
> Section - Subsection, prerelease fix
- Describe a change for adding it to https://github.com/JetBrains/compose-multiplatform/blob/master/CHANGELOG.md
```

Reply:
Or:
```
prerelease fix - add it if it fixes a bug introduced in a previous prerelease version (dev/beta). It will be excluded from the stable changelog
> Section - Subsection
- _(prerelease bug)_ Describe a change for adding it to https://github.com/JetBrains/compose-multiplatform/blob/master/CHANGELOG.md
```
As we did in https://github.com/JetBrains/compose-multiplatform/releases/tag/v1.6.0-beta02

Reply:
Okay, I decided to keep it for now, but added a comment

Reply:
Let's keep `- _(prerelease fix)_ ` mark for now - separate category takes too much space

Reply:
Done